<launch>
    <node name="fake_point_pub" type="staticFP.py" pkg="cyber_security" />

</launch>
